source/includes/data-formats/time-series.cpp

```diff
@@ -0,0 +1,63 @@
+#include <iostream>
+#include <chrono>
+
+#include <bsoncxx/builder/list.hpp>
+#include <bsoncxx/builder/stream/document.hpp>
```

Comment:
```suggestion
#include <bsoncxx/builder/basic/document.hpp>
```

Suggest replacing includes. This example does not use the `bsoncxx::builder::list` or `bsoncxx::builder::stream`, but does use `bsoncxx::builder::basic`.